media-sound/wavpack/wavpack-5.0.0.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="Open audio compression format"
```

Comment:
I'd keep the format part, but I'd say "format & tools". After all, it's a format first, I think.